### PR TITLE
Enabled  isteps proc_clock_test, set_ref_clock, sbe_config_update

### PR DESCRIPTION
**Pull request by ojayanth**
- Enabled  isteps proc_clock_test , set_ref_clock, sbe_config_update
 - Replaced  CFAM based istep mode settings to ipl function.

### CODE REVIEW

File: src/dll/edbgEcmdDll.C [hunk 1]
```diff
@@ -2078,7 +2031,7 @@ uint32_t dllIStepsByName(std::string i_stepName) {
        if (!rc)
        {
            //Set IPL mode to interactive
-           rc = setIplMode(IPL_MODE_ISTEP);
+           ipl_init(IPL_DEFAULT);
```
Comment (lkammath) [inline src/dll/edbgEcmdDll.C:2034]: rc =

Comment (ojayanth) [reply src/dll/edbgEcmdDll.C]: done

**Comment by ojayanth** [comment]
Please hold the merge, testing going on,

**Comment by lkammath** [comment]
Please refer to PR #40 . This can be closed

**Pull request closed without merging**